@ndlib/gatsby-theme-marble/src/components/App/Pages/User/UserBody/PortfolioList/index.js

```diff
@@ -11,17 +11,32 @@ import { COMPILATIONS_LISTING_PAGE } from 'store/actions/displayActions'
 import { isLoggedIn, ownsPage } from 'utils/auth'
 import style from './style.module.css'
 import MaterialButton from 'components/Internal/MaterialButton'
-import ActionModal from 'components/Internal/ActionModal'
-import DangerDelete from 'components/App/Pages/Portfolio/PortfolioBody/PortfolioEditSettings/PortfolioSettingsContent/DangerDelete'
+import { deleteData } from 'utils/api'
+import { navigate } from 'gatsby'
 
 const PortfolioList = ({
   user,
   loginReducer,
 }) => {
+  const beGone = (portfolio) => {
+    const areYouSure = window.confirm('Are you sure you want to delete this protfolio?') ? (
+      deleteData({
+        loginReducer: loginReducer,
+        contentType: 'collection',
+        id: portfolio.uuid,
+        successFunc: () => {
+          navigate(`/user/${loginReducer.user.userName}`)
```

Comment:
Since this action is being called from the `/user/{user.userName}` page, navigation here won't actually do anything. After the item is deleted the page will not refresh and the deleted portfolio will still be visible. We're not using context here like we probably should be, but I think we can still make this work without the hassle of adding that in. 

Let's try changing:
```
        successFunc: () => {
          navigate(`/user/${loginReducer.user.userName}`)
        },
```
 to
```
        successFunc: () => {
          setPortfolios(portfolios.filter(p => {
            p.uuid !== portfolio.uuid
          })
        },
```
**Note: The above is not tested, so it may need a couple of tweaks.**